mlflow/__init__.py

```diff
@@ -47,6 +47,7 @@
 from mlflow.utils.logging_utils import _configure_mlflow_loggers
 
 # Lazily load mlflow flavors to avoid excessive dependencies.
+anthropic = LazyLoader("mlflow.gemini", globals(), "mlflow.anthropic")
```

Comment:
```suggestion
anthropic = LazyLoader("mlflow.anthropic", globals(), "mlflow.anthropic")
```
just confirming if this is a typo

Reply:
that's a typo, thanks for spotting!